js_tests/admin/RelatedObjectLookups.test.js

```diff
@@ -1,6 +1,7 @@
 module('admin.RelatedObjectLookups');
 
 test('html_unescape', function(assert) {
+    'use strict';
```

Comment:
Since these are all test files, I think all these `'use strict'` statements at the top of each function could be replaced with a single `'use strict'` at the top of each file.